src/api/stops/index.ts

```diff
@@ -0,0 +1,16 @@
+import stopData from './data.json';
+
+export interface IStop {
+  id: string;
+  name: string;
+  description: string;
+  // GPS Location of the stop
```

Comment:
total nitpicking: `// Coordinates of stop` is technically more accurate. or ` WGS84/Pseudo-Mercator coordinates of stop`. Preference for the latter. If there is going to be a map, the wrong coordinate system can be used to create coordinate values and can waste dev time trying to figure out why their points are wrongly located on the map. Of course if you are using Google Maps to get your coordinates or any major online map, you will likely get coordinates in the right system, but its not a 100% guarantee. I like to be explicit.

Reply:
Nice! Never even thought about that.